giraph-core/src/main/java/org/apache/giraph/bsp/CentralizedServiceWorker.java

```diff
@@ -256,4 +256,19 @@ void cleanup(FinishedSuperstepStats finishedSuperstepStats)
    */
   void addressesAndPartitionsReceived(
       AddressesAndPartitionsWritable addressesAndPartitions);
+
+  /**
+   * Send the counter values to the master after every superstep
+   * and also after all supersteps are done. This is called before closing
```

Comment:
These comments don't describe exactly what the method does. We're not really sending to the master.